systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectBuilderST.java

```diff
@@ -94,6 +94,8 @@ class ConnectBuilderST extends AbstractST {
     private static final String PLUGIN_WITH_OTHER_TYPE_NAME = "plugin-with-other-type";
     private static final String PLUGIN_WITH_MAVEN_NAME = "connector-from-maven";
 
+    private static final String SHARED_KAFKA_CLUSTER = "shared-kafka-connect";
```

Comment:
so is it Kafka cluster or KafkaConnect cluster?

Reply:
Kafka cluster, good catch thanks